strawberry/types/type_resolver.py

```diff
@@ -385,7 +313,8 @@ def _get_fields(cls: Type) -> List[FieldDefinition]:
                 default_value=getattr(cls, field.name, undefined),
             )
 
-        field_name = cast(str, field_definition.origin_name)
+        field_name = field_definition.origin_name
+        assert field_name is not None  # Not Optional by this point
```

Comment:
can we add a comment for the assertion here?

```
assert field_name is not None, "This happens when..."
```

Reply:
I have an in-line one. Would you like it on its own line, or are you wanting a more detailed comment?

Reply:
A more detailed comment could be better, but I was mainly referring to add a message to the assertion (I used the wrong wording 😊), something like this:

<img width="285" alt="image" src="https://user-images.githubusercontent.com/667029/99309987-349bff00-2852-11eb-937e-626362d722fd.png">

Reply:
Ah ok, sure